src/App.js

```diff
@@ -1,26 +1,199 @@
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React in MateAcademy!
-        </a>
-      </header>
-    </div>
+import './fonts/stylesheet.css'
+import './css/App.css';
+
+import TodoItem from './Components/TodoItem';
+import Footer from './Components/Footer';
+
+class App extends React.Component{
+  constructor(){
+    super()
+    this.state = {
+      inputValue: '',
+      tasks: [],
+      activeTasks: [],
+      completedTasks: [],
+      item: 0,
+      clickOnFooter: false
+    }
+
+    this.addTask = this.addTask.bind(this)
+    this.inputTask = this.inputTask.bind(this)
+    this.handleChange = this.handleChange.bind(this)
+    this.removeTask = this.removeTask.bind(this)
+    this.showActive = this.showActive.bind(this)
+    this.showCompleted = this.showCompleted.bind(this)
+    this.showAll = this.showAll.bind(this)
+    this.clearCompleted = this.clearCompleted.bind(this)
+    this.handleChangeFooter = this.handleChangeFooter.bind(this)
+
+
+  }
+
+  inputTask(event) {
+    this.setState({inputValue: event.target.value});
+  }
+
+  addTask(event) {
+    if(event.keyCode === 13){
+      const {inputValue, item} = this.state;
+      this.setState(prevState => {
+        if(inputValue === '')return;
+          const copy = prevState.tasks.concat({})
+          let count = prevState.item + 1
+          copy[item]['text'] = inputValue;
+          copy[item]['checked'] = false;
+          copy[item]['id'] = item
+          return {
+             inputValue:'',
+             tasks: copy,
+             activeTasks: copy,
+             item: count
+           };
+        })
+      }
+    }
+
+  removeTask(index) {
+    this.setState(prevState => {
+      let count = prevState.item;
+      const copy = [...prevState.tasks]
+      if(copy[index].checked === false){
+        count--;
+      }
+       copy.splice(index,1);
+       const updateActiveTask =  this.serchActivTask(copy);
+       const updateCompletedTask = this.updateCompletedTask(copy)
+
+
+
+      return {tasks: copy,
+              item: count,
+              activeTasks: updateActiveTask,
+              completedTasks: updateCompletedTask
+              }
+    })
+
+  }
+
+
+serchActivTask(arr){
+  const filtredArr = arr.filter((el) => {
+      return el.checked === true ? false : true
+      })
+      return filtredArr;
+}
+
+updateCompletedTask(arr){
+  const filtredArr = arr.filter((el) => {
+      return el.checked === true ? true : false
+      })
+      return filtredArr;
+}
+
+
+  handleChange(id) {
+    this.setState(prevState => {
+      let count = prevState.item;
+        const updatedTask = prevState.tasks.map(task => {
+            if (task.id === id) {
+                task.checked === false ? count --: count ++;
```

Comment:
```suggestion
                task.checked === false ? count--: count++;
```
I strongly recommend you to install eslint and format your code properly

Reply:
Also, I believe you can get rid of `count` and simplify your code a lot